libs/text-splitters/tests/integration_tests/test_text_splitter.py

```diff
@@ -118,3 +119,83 @@ def test_sentence_transformers_multiple_tokens(sentence_transformers: Any) -> No
         - splitter.maximum_tokens_per_chunk
     )
     assert expected == actual
+
+
+@pytest.mark.requires("langchain")
+def test_text_splitter_from_tiktoken_start_index() -> None:
+    from langchain.docstore.document import Document
```

Comment:
@ccurme I haven't yet come across other tests that import from a different langchain package like this, so I'm not sure how I should go about this. Can you point me to an example as a reference point if you know of any?

Reply:
Maybe it would be better to move these tests to the langchain package since they rely on both `langchain.docstore.document` and `langchain_text_splitters`.

Reply:
Scratch that. I just noticed that `langchain-core` is already a test dependency. Sorry for the noise!